frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -112,7 +112,7 @@ export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses,
         )}
         {!hasIssues && (isWaitingOrDisabled || isError) && (
           <ExternalLink
-            href="https://docs.openshift.com/container-platform/latest/support/getting-support.html"
+            href="https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.html"
```

Comment:
You should use `openshiftHelpBase` to link to the correct edition and version of the docs for the cluster.

https://github.com/openshift/console/blob/master/frontend/public/components/utils/documentation.tsx#L2

Reply:
@spadgett thanks, edited